proposals/html-modules-explainer.md

```diff
@@ -124,6 +124,16 @@ This example demonstrates how the above custom element definition can be consume
 
 [main.html](demo/main.html)
 
+## Major open design questions
+
+The following list is **not** comprehensive of all open issues; the intent here is to highlight questions that could have a significant impact on the shape of the final design depending on the way they are decided.
+
+- Should HTML Modules use a new MIME type, or the existing `text/html`? [[1](https://github.com/w3c/webcomponents/issues/742)]
+- How should HTML modules interact with CSP?  In particular, should their inline scripts be allowed to run even without an `unsafe-inline` directive? [[1](https://github.com/w3c/webappsec/issues/544)], [[2](https://discourse.wicg.io/t/proposal-html-modules/3309/2)]
+- Should non-`type="module"` scripts in an HTML module yield an error or be implicitly interpreted as module scripts? [[1](https://github.com/w3c/webcomponents/issues/798)]
+- Should there be a declarative way to consume HTML modules?
```

Comment:
This is an interesting issue that I didn't know about because I don't follow MSEdgeExplainers (I will now!). Is the intent to open an issue here about that subject or is your team debating it amongst yourselves first?

Reply:
You're welcome to comment on that issue there (no need to duplicate issues). We prefer new issues be opened here (or in whatwg/html) from now on though :)

Reply:
Ok, taking the discussion there then.

Reply:
I'd prefer if it was moved. We have standards forums for a reason.